docker/registry/internal/acr.go

```diff
@@ -43,19 +43,25 @@ func (c *azureContainerRegistry) WrapTransport(...TransportWrapper) error {
 
 // Tags fetches tags for an OCI image.
 func (c azureContainerRegistry) Tags(imageName string) (versions tools.Versions, err error) {
-	apiVersion := c.APIVersion()
-
 	// acr puts the namespace under subdomain.
-	if apiVersion == APIVersionV1 {
-		url := c.url("/repositories/%s/tags", imageName)
-		var response tagsResponseV1
-		return c.fetchTags(url, &response)
-	}
-	if apiVersion == APIVersionV2 {
-		url := c.url("/%s/tags/list", imageName)
-		var response tagsResponseV2
-		return c.fetchTags(url, &response)
-	}
-	// This should never happen.
-	return nil, nil
+	url := c.url("/%s/tags/list", imageName)
```

Comment:
So ACR is always api v2 now? Even for anonymous access?

Reply:
All the providers are using v2 now.